### PR TITLE
Add Dishwasher Device Type

### PR DESCRIPTION
**Pull request by mideayanghui**
📖 Description
This PR adds  new device type for Matter 1.2:
   Dishwasher

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 1]
```diff
@@ -1944,6 +1944,31 @@ limitations under the License.
...
+        <name>MA-dishwasher</name>
+        <domain>CHIP</domain>
+        <typeName>Matter Dishwasher</typeName>
+        <profileId editable="false">0x0103</profileId>
+        <deviceId editable="false">0x0075</deviceId>
+        <clusters lockOthers="true">
+            <include cluster="Identify" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>IDENTIFY_TIME</requireAttribute>
+                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
+                <requireCommand>Identify</requireCommand>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1957]: Why are these "require*" bits here?

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml [hunk 2]
```diff
@@ -1944,6 +1944,31 @@ limitations under the License.
...
+            </include>
+            <include cluster="On/Off" client="false" server="false" clientLocked="true" serverLocked="false">
+                <requireAttribute>ON_OFF</requireAttribute>
+                <requireCommand>Off</requireCommand>
+                <requireCommand>On</requireCommand>
+                <requireCommand>Toggle</requireCommand>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/matter-devices.xml:1963]: Again, not sure why these require* bits are here.  What do they do?

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 1 above)
Comment (mideayanghui) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml]: Remove it.

File: src/app/zap-templates/zcl/data-model/chip/matter-devices.xml (hunk 2 above)
Comment (mideayanghui) [reply src/app/zap-templates/zcl/data-model/chip/matter-devices.xml]: Remove it.

**Review by andy31415 — APPROVED** [review]

**Pull request merged**